tests/framework/extensions/provisioning/verify.go

```diff
@@ -420,24 +445,52 @@ func VerifySnapshots(client *rancher.Client, localclusterID string, clusterName
 	return snapshotToBeRestored, err
 }
 
-// getSnapshots is a helper function to get the snapshots for a cluster
-func GetSnapshots(client *rancher.Client, localclusterID string, clusterName string) ([]string, error) {
+// GetRKE1Snapshots is a helper function to get the existing snapshots for a downstream RKE1 cluster.
+func GetRKE1Snapshots(client *rancher.Client, clusterName string) ([]string, error) {
```

Comment:
if this is only used here, then can you make it a private function? Otherwise if it is public, I feel this should live in the extensions/etcdsnapshot file instead as it isn't validating anything.

Reply:
RBAC testing uses the now RKE2K3S version, so it was made public in that situation I'm pretty sure. I think for RKE1 it should remain public for the same reason.

Reply:
I can move them over to the etcdsnapshot extension, that makes sense to me.